fastlane/Fastfile

```diff
@@ -171,8 +171,22 @@ ENV["HAS_ALPHA_VERSION"]="true"
     android_finalize_prechecks(options)
     hotfix = android_current_branch_is_hotfix
     android_update_metadata(options) unless hotfix
-    android_bump_version_final_release() unless hotfix
+
+    # Run release checks
+    project_folder = Dir.pwd + "/.."
```

Comment:
It would be nice to avoid having to pass this parameter – instead, if we could determine it automatically, it'd keep the code cleaner and require less developer overhead to determine :)

Reply:
Thank you!  I will give look into this! 🙂

Reply:
now having the actions figure out the project directory and no longer requiring it to be passed in.  The change is made here: 377ef5855cd08d90fee91a28c72ac2c599686e6d

Reply:
see this [PR ](https://github.com/wordpress-mobile/release-toolkit/pull/96) in the `release-toolkit` repo for the action changes.